config/schema.json

```diff
@@ -357,37 +358,38 @@
         },
         "lanAddress": {
           "description": "LAN address and port for the gateway. If not specified, defaults to the task/node address.",
-          "type": "object",
+          "type": ["object", "null"],
```

Comment:
Question: I'm not clear on why `null` is defined as a valid type when the fields themselves aren't required. Is this to allow for the possibility to define the config like `"lanAddress": null`?

Reply:
Yeah, it's kind of silly, but I setup the config schema this way because it slightly simplifies passing fields from the Terraform module. We don't have to worry in Terraform whether a field is `null` or typed (string, list, int) for it to pass validation, since there are so many fields we don't specify by default.

It may not make as much sense for some of these fields. And we've stopped using null so much for default values in the Terraform module, so maybe we could rethink this.